Makefile

```diff
@@ -165,21 +158,22 @@ docker-build:
 docker-push: ## Push docker image with the manager.
 	docker push ${IMG}
 
-kind-load-image:
-	kind load docker-image --name $(KIND_CLUSTER) ${IMG}
+kind-single-e2e-test: single-create single-prepare e2e-test
+
+kind-multi-e2e-test: multi-create multi-prepare e2e-test
 
-kind-single-e2e-test: single-up e2e-test
+single-create: cleanup create-kind-cluster cert-manager
```

Comment:
The philosophy is:

* The -create targets should typically be executed only once
* The -prepare targets should typically be executed each time we want to re-run an e2e test.

Reply:
nice